types/angular/index.d.ts

```diff
@@ -1390,10 +1390,10 @@ declare namespace angular {
 
     interface IControllerService {
         // Although the documentation doesn't state this, locals are optional
-        <T>(controllerConstructor: new (...args: any[]) => T, locals?: any, later?: boolean, ident?: string): T;
-        <T>(controllerConstructor: Function, locals?: IControllerLocals, later?: boolean, ident?: string): T;
-        <T>(controllerConstructor: Function, locals?: any, later?: boolean, ident?: string): T;
-        <T>(controllerName: string, locals?: any, later?: boolean, ident?: string): T;
+        <T>(controllerConstructor: new (...args: any[]) => T, locals?: any): T;
+        <T>(controllerConstructor: (...args: any[]) => T, locals?: IControllerLocals): T;
```

Comment:
Please see the last [common mistake](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes). While we don't have union types here, it's the same situation. The next signature differs from this one only in the type of the `locals` argument. So I think this one should be removed as it does nothing.